### ISSUE TITLE
Under Bridge Tiles

### ISSUE THREAD

**Issue by kevin-shannon** [Bl4ckSh4rk/PKHeXFeebasLocatorPlugin#2]
This model seems to be missing a part of the puzzle with the tiles that are under the bridge. On my physical copy of the game all 10 tiles under the bridge happen to spawn feebas.  [This post](https://old.reddit.com/r/ProfessorOak/comments/bwitoa/feebas_tiles_to_fish_into_rubysapphire/)
claims that it's computed as one tile which one make sense and calls it 132 which is one of your yellow tiles. I tried changing the seed to spawn on 132 but I didn't have any luck. Could we investigate this to see if it really is 132 or maybe another unaccounted for tile?

**Comment by Bl4ckSh4rk** [comment]
Sorry for the late reply. Github doesn't seem to notify me about issues ...

There's a German debug version of Pokemon Ruby which has a nice menu that shows on which tiles you can actually find Feebas (Encounter location) and which tile is in front of you. According to that any tile under the bridge is 448, not 132. Like any other tile that isn't a water tile.
![Pokemon Rubin (Debug)](https://user-images.githubusercontent.com/25778475/79478796-2233bb80-800c-11ea-9bc8-bf5c9049d234.png)
The algorithm to determine Feebas tiles has been dumped from the ROM ([see here](https://www.smogon.com/forums/threads/past-gen-rng-research.61090/page-34#post-3986326)). It does not result in 448 for any seed. So fishing Feebas under the bridge is impossible.

**Comment by kevin-shannon** [comment]
Thank you for the debug tool I've been looking for something like that for a while. As for it being impossible, I have caught feebas under the bridge so it is definitely possible it must be that the there is something more going on than what's just in the dump of the Feebas tiles algorithm. Can we be sure that the reconstruction is a 1:1 copy of the original?

**Comment by Bl4ckSh4rk** [comment]
Well, Tux provided a disassembly of the games code which he ported into C++. I think the code is correct but I'm beginning to think that the debug Ruby might be wrongly calling the tiles 448 while it's actually another tile. 132 or whatever.

When you have caught Feebas under the bridge which seed did you have? A seed where it should have been on 132?

**Comment by kevin-shannon** [comment]
I wish I knew the seed but atlas I discovered this while playing on my physical emerald cartridge that I've had since the game was first released. Since my internal battery had died the seed did not change so I was able to reliably catch feebas on all tiles under the bridge everyday.

**Comment by Bl4ckSh4rk** [comment]
~~Ok. I'm going to investigate this but it will take a while.~~

So I just tried a seed where Feebas should be on tile 132 and indeed I was able to encounter it under the bridge! Just in case I have checked the other inaccessible tiles as well but they don't result in Feebas beeing under the bridge.

Nice find! Many thanks for letting me know about this! I'm going to update the plugin to also include the bridge.
